data/coalition/coalition missions.txt

```diff
@@ -2335,3 +2335,469 @@ mission "Coalition: Submarines 4"
+			`	Sedlitaris ties a rope around the end of the pylon that's visible and pulls it, forcing up the dirt as she brings the rest of the device to surface. She connects it with the pylon you brought, and accesses the data from there. "Much more active here, the expedition was," she says. "Three recordings, there are. Bring these back to your ship, we should, listen to the first one there, we can."`
+			`	Both of you carrying a pylon, you make your way out of the forest and back to your ship, where Sedlitaris immediately plays the oldest entry: "Log started. Send two Punishers further north, I did, but returned yet, they have not. As for the crimson anomaly on the nearby western system, still unresponsive, it is. A reaction from it, nothing got, be it a Finisher barrage or attempting to ram it. A simple nebula, the researchers claim it is not, think that we are missing something, they do. Wait a few more days for the Punishers to return, I will. In the meantime, back to that system, I will go. Maybe learn something about the anomaly, we can."`
+			`	After Mimodum's voice has disappeared, Sedlitaris urges you to take off. "Heard of this 'anomaly' before, I had not. Look for the system it's in, at once we must. Hopefully still there after the centuries, it is."`
+				accept
+	to complete
+		has "Coalition: Expeditions Past 7B: failed"
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`
+
+mission "Coalition: Expeditions Past 7B"
+	invisible
+	landing
+	to offer
+		has "Coalition: Expeditions Past 6: done"
+	on enter "Terminus"
+		conversation
+			`Sedlitaris fiddles with the pylon you recovered from <origin>, and lets the second recorded log play while she gazes out to see the "crimson anomaly" Mimodum had mentioned.`
+			`	"Tried just about everything now, we have. A finisher barrage, bombardment from afar, driving asteroids through it, even activating our jump drive while immersed in the red cloud. Told me that it shows some signs of rotation, the chief researcher has, tried to show me the axis around which it spins, she did. Whatever the case, find anything worthwhile about the cloud, we could not. Returning to Summit now, we are, and from there to regroup with the fleet."`
+			`	The recording ends, but Sedlitaris continues looking out to the "cloud" for a few minutes, in silence.`
+			choice
+				`	"Are you going to play the last one?"`
+					goto last
+				`	"Is something wrong?"`
+			`	"No, just... taking in the view, I was. Unlike any phenomena from within the Coalition, this is."`
+			label last
+			`	She moves back to the pylon, and hesitates for a moment. "Back to Summit, Mimodum said they would go. Better understand his next steps, we would, if listen to the last entry on that clearing where we found this, we did," she says, tapping the pylon.`
+			`	You tell her you'll go back to <origin> soon, and she heads to her bunk.`
+		fail
```

Comment:
I haven't tested that, will need to check.

Reply:
Tested it. Doesn't seem to work.
Thing is the mission automatically offers and accepts alongside the main one ("Curious Red Cloud"), so `decline` just forces an endpoint to the conversation. The mission was already accepted when it offered, it can't be declined here.

I'm not sure what you meant about altering the conditions for the other missions (assuming you meant missions past missions 7/7B), but I don't think `decline` would work because of the same reason, the missions were already accepted, so can't decline them afterwards.

Reply:
Got it. what I meant be altering conditions for others was just updating everything so the relevant conditions were just replaced with `declined` instead of `failed`. But since this fix doesn't work here that's not relevant.